src/unconstrained/de.cpp

```diff
@@ -117,7 +117,11 @@ optim::internal::de_impl(
 #ifdef OPTIM_USE_OPENMP
     #pragma omp parallel for num_threads(omp_n_threads) firstprivate(rand_vec)
 #endif
+#ifdef _MSC_VER
```

Comment:
This is an ugly code change. I suggest using an alias that depends on the build system.

Reply:
Alright, this is done now (see [commit (effb64)](https://github.com/kthohr/optim/pull/54/commits/effb64f01d44930c52013c7d0ee879e950e81ea6) ).